nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss update linkscroll 94

# Description - updated Navbar Typing Text, scroll stopping location for desktop and mobile, 

## Type of change

- [x] General updates on Navbar (non-breaking change which fixes an issue)

# Testing Instructions
Fetch branch ss-update-linkscroll-94, checkout into it and npm start to serve. Check links by clicking the sections in the navbar and see if it scrolls to the correct location in the website. Also check typing text in navbar to see if it retypes

# Checklist:
